website2/docs/schedulers-k8s-with-helm.md

```diff
@@ -50,55 +50,52 @@ $ brew install kubernetes-helm
 You can install Helm on Linux using a simple installation script:
 
 ```bash
-$ curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get > install-helm.sh
-$ chmod 700 install-helm.sh
-$ ./install-helm.sh
+$ curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+$ chmod 700 get_helm.sh
+$ ./get_helm.sh
 ```
 
-## Installing Helm in your Kubernetes cluster
+## Installing Heron on Kubernetes
 
-To run Helm on Kubernetes, you need to first make sure that [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) is using the right configuration context for your cluster. To check which context is being used:
+To use Helm with Kubernetes, you need to first make sure that [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) is using the right configuration context for your cluster. To check which context is being used:
 
 ```bash
 $ kubectl config current-context
 ```
 
-If the context is correct, then you can get Helm running using just one command:
+Once you've installed the Helm client on your machine and gotten Helm pointing to your Kubernetes cluster, you need to make your client aware of the `heron-charts` Helm repository, which houses the chart for Heron:
 
 ```bash
-$ helm init
+$ helm repo add heron-charts https://storage.googleapis.com/heron-charts
```

Comment:
I don't think this google bucket exists anymore.  I wonder if we should modify the instructions to build and deploy from the source.  Thoughts?

Reply:
That's not a bad idea. I had to figure it out on my own and having the `bazel build` target to `generate-helm-charts` documented would be nice.

Reply:
yeah I agree.   I'll wait to see if anyone else has any feedback.